### DISCUSSION TITLE
AttributeError: 'Mesh' object has no attribute 'calc_normals_split'

### DISCUSSION THREAD

**Discussion by Mrtn1225** [StjerneIdioten/I3D-Blender-Addon#206] [Help] [answered]
This is the first time I use blender and whatever I try to export anything to i3d this is the message that I get at the end of a log, I've tried fixing this for hours to no avail, How do I fix this?

Here is the rest of the log:
![Posnetek zaslona (5699)](https://github.com/StjerneIdioten/I3D-Blender-Addon/assets/165223315/f700b831-0539-4416-90ee-6afb7ccf4b69)

**Comment by StjerneIdioten** [comment, marked as answer]
I believe this is because Blender 4.1.0 changed something. If you switch back to an earlier Blender version like 4.0.0, it will probably work. Or you can wait for me to merge #205, which I believe is a fix for this :-)

**Reply by Mrtn1225** [reply]
Thank you very much for the help and answer, now it works like it should.
